### DISCUSSION TITLE
New embeddings, pdf styling and newspaper3k support 👨‍👨‍👦‍👦

### DISCUSSION THREAD

**Discussion by assafelovic** [assafelovic/gpt-researcher#397] [Announcements]
Another great release thanks to the amazing community! ❤️

Big shoutout to the following contributions:
@proy9714 for adding newpaper3k support for better article scraping https://github.com/assafelovic/gpt-researcher/pull/365
@jimmylin0979 for adding support for additional embeddings such as Mistral, Ollama and HuggingFace https://github.com/assafelovic/gpt-researcher/pull/375
@assafelovic adding support for pdf styling of research reports https://github.com/assafelovic/gpt-researcher/pull/396
@WarrenTheRabbit for fixing a documentation typo https://github.com/assafelovic/gpt-researcher/pull/391

Thank you to everyone and looking forward for more contributions!

<hr /><em>This discussion was created from the release <a href='https://github.com/assafelovic/gpt-researcher/releases/tag/v0.1.5'>New embeddings, pdf styling and newspaper3k support 👨‍👨‍👦‍👦</a>.</em>

**Comment by WarrenTheRabbit** [comment]
What an unexpected (but lovely) mention. Thank you, @assafelovic.
